lib/internal/streams/util.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
```

Comment:
I would prefer we avoid adding one more file in for this. Maybe we can move it in `legacy` and call it something else?

Reply:
It does not really matter. We can call it `base`.

Adding more files on streams will slow down bootstrap.

Reply:
> It does not really matter. We can call it base.

? Not sure I follow. If we put it in legacy it will end up public. Or is that what you mean doesn't matter?

Reply:
We can change the export of `legacy.s` file and export `{ Stream, prependListener }`.

Reply:
Fixed